cps/db.py

```diff
@@ -957,7 +957,18 @@ def get_cc_columns(self, config, filter_config_custom_read=False):
     def get_search_results(self, term, config, offset=None, order=None, limit=None, *join):
         order = order[0] if order else [Books.sort]
         pagination = None
-        result = self.search_query(term, config, *join).order_by(*order).all()
+        
+        # search also through the subtitles (for videos)
+        other_terms = lb_search.get_search_terms(term)
+        # lb_search.get_search_terms returns a list of video titles, "term" parameter is expected to be a book/video title
+        term = [term] + other_terms
```

Comment:
At the end they both are.

Here is how this starts:
term = "some_value"
other_terms = [item1, item2, item3...]

As you can see, term is not a list yet, it's a string. I have to convert it into a list. I do so by using a list literal (`my_list = [string]`).

Since I want a single list that includes other_terms (which is already a list), I do it in on the fly.
```
term = [term] + other_terms